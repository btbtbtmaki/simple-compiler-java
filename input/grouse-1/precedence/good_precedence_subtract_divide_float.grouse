main {

  print 8.6 - 8.7 / 6.2 nl;
  print 1.6 - 5.8 / 1.4 nl;
  print 2.0 - 6.1 / 6.4 nl;
  print 5.6 - 6.4 / 6.5 nl;
  print 8.8 - 5.5 / 1.1 nl;
  print 5.2 - 4.4 / 4.7 nl;
  print 3.2 - 6.3 / 8.8 nl;
  print 2.2 - 9.5 / 3.1 nl;
  print 4.0 - 6.3 / 9.5 nl;
  print 7.4 - 1.1 / 2.2 nl;
  print 5.5 - 8.8 / 1.5 nl;
  print 7.5 - 1.1 / 8.8 nl;

  print 8.3 / 3.1 - 5.8 nl;
  print 4.3 / 6.6 - 7.8 nl;
  print 7.7 / 5.9 - 7.3 nl;
  print 2.1 / 3.2 - 6.7 nl;
  print 2.7 / 2.4 - 7.2 nl;
  print 5.5 / 6.7 - 8.4 nl;
  print 1.8 / 1.1 - 9.9 nl;
  print 2.4 / 8.4 - 5.4 nl;
  print 5.7 / 6.6 - 2.1 nl;
  print 7.3 / 5.9 - 5.0 nl;
  print 7.6 / 3.6 - 3.1 nl;
  print 7.4 / 3.9 - 7.5 nl;

  print 6.5 - 7.6 / 1.6 - 7.3 nl;
  print 6.7 - 2.2 / 3.3 - 5.0 nl;
  print 3.1 - 2.9 / 3.2 - 3.1 nl;
  print 4.6 - 9.7 / 5.4 - 7.0 nl;
  print 5.4 - 7.2 / 7.8 - 7.2 nl;
  print 9.9 - 4.2 / 7.5 - 6.8 nl;
  print 5.5 - 7.1 / 5.1 - 2.0 nl;
  print 3.3 - 2.6 / 7.2 - 6.8 nl;
  print 5.2 - 3.5 / 7.1 - 9.9 nl;
  print 1.2 - 3.4 / 7.3 - 7.0 nl;
  print 4.3 - 7.2 / 1.3 - 4.2 nl;
  print 7.8 - 8.6 / 7.7 - 3.2 nl;

  print 9.5 / 2.8 - 4.7 / 9.5 nl;
  print 7.2 / 8.6 - 4.2 / 5.6 nl;
  print 1.4 / 5.5 - 5.9 / 9.9 nl;
  print 9.5 / 1.2 - 1.7 / 4.9 nl;
  print 1.4 / 9.0 - 2.9 / 2.8 nl;
  print 2.7 / 8.4 - 3.9 / 5.6 nl;
  print 9.2 / 6.4 - 2.2 / 7.5 nl;
  print 4.8 / 6.5 - 2.8 / 9.0 nl;
  print 3.6 / 7.0 - 4.0 / 3.6 nl;
  print 4.3 / 5.3 - 4.0 / 6.2 nl;
  print 1.9 / 7.2 - 1.8 / 1.0 nl;
  print 5.6 / 8.0 - 5.0 / 2.0 nl;

}

