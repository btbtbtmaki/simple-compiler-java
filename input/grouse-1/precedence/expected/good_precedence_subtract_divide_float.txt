7.19677
-2.54286
1.04688
4.61538
3.8
4.26383
2.48409
-0.864516
3.33684
6.9
-0.366667
7.375
-3.12258
-7.14848
-5.99492
-6.04375
-6.075
-7.5791
-8.26364
-5.11429
-1.23636
-3.76271
-0.988889
-5.60256
-5.55
1.03333
-0.90625
-4.1963
-2.72308
2.54
2.10784
-3.86111
-5.19296
-6.26575
-5.43846
3.48312
2.89812
0.0872093
-0.341414
7.56973
-0.880159
-0.375
1.14417
0.42735
-0.596825
0.166159
-1.53611
-1.8
